api/v1beta1/horizontest_types.go

```diff
@@ -28,6 +29,11 @@ import (
 type HorizonTestSpec struct {
 	CommonOptions `json:",inline"`
 
+	// The desired amount of resources that should be assigned to each test pod
+	// spawned using the HorizonTest CR. https://pkg.go.dev/k8s.io/api/core/v1#ResourceRequirements
+	// +kubebuilder:default:={limits: {cpu: "2000m", memory: "4Gi"}, requests: {cpu: "1000m", memory: "2Gi"}}
+	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
@ashu-011, can you please :pray: suggest the correct Resource Limits for the horizon test pods? It will be possible to override the default value in the HorizonTest CR.